src/components/CompanyPlacement/CompanyPlacement.tsx

```diff
@@ -40,7 +62,9 @@ function CompanyPlacement() {
         </p>
       </div>
       <div className={imageContainer}>
-        <PlacementsTable images={images} />
+        {placementsWithImage.map(obj => (
```

Comment:
Challenge: can we add a ‘order’ property to the JSON so that we can control the order, instead of alphabetical filename handling? I think the preferred order would be bank - asset management - pension - tech/other

Reply:
A hacky solution might be to prefix the JSON files with numbers indicating the order, eg 1-banks.json that will trick the query to return it first I think